### ISSUE TITLE
ComPact can be used/referenced in .NET Standard

### ISSUE THREAD

**Issue by eNeRGy164** [bartschotten/com-pact#1]
It's not possible to use (parts of) ComPact from a .NET Standard library.

Quick testing shows that changing the NuGet references:
```diff
- Microsoft.AspNetCore
- Microsoft.AspNetCore.Hosting
+ Microsoft.AspNetCore.Hosting.Abstractions
+ Microsoft.AspNetCore.Http.Abstractions
```
almost resolves everything, except for the code actually using the WebHost. Maybe you can isolate this code in a separate NuGet package?

Also, as a library you might considering lowering the dependencies to the lowest possible version (f.e. 2.1.0 instead of 2.2.7) to prevent forcing consumers to upgrade to at least those versions when they want to use your project. (Basically only require higher versions if bugfixes/used features require this)

**Comment by bartschotten** [comment]
Downgraded dependencies somewhat in 4555ee533c72986b61fb3fd42fe7b2de7fae07cf.

Removed RestSharp dependency entirely in 4555ee533c72986b61fb3fd42fe7b2de7fae07cf

**Comment by bartschotten** [comment]
Solved in 9bd64dec214f3b53a848b3ab2f48c4e1faae2ff2
